pyproject.toml

```diff
@@ -53,6 +53,62 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
+alignment = [
+    "dtw-python",
+    "numba",
+]
+annotation = [
+    "attrs",
+    "hmmlearn",
+    "numba",
+    "pyod",
+]
+classification = [
+    "esig",
+    "mrsqm",
+    "numba",
+    "tensorflow",
+    "tsfresh",
+]
+clustering = [
+    "numba",
+    "tslearn",
+]
+forecasting = [
+    "hcrystalball",
```

Comment:
I would remove `hcrystalball` and `scikit-optimize` for maintenance reasons

Reply:
Any suggestions? New sub-extra, or complete removal?

Reply:
complete removal would be my suggestion - as you said, we should avoid strong restrictions on bounds